refactor/main.go

```diff
@@ -0,0 +1,87 @@
+package main
```

Comment:
FYI you might like to have a read about [`sed`](https://man7.org/linux/man-pages/man1/sed.1.html) which is a unix command that does this sort of thing. 😇

    find -name '*.go' | xargs sed -i 's/find-regex/replacement/'